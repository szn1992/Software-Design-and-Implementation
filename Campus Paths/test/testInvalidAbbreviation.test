# Test the output when invalid abbreviated names are entered
# when getting the shortest path

r
OOG
CSE
r
WHAT
AH
q