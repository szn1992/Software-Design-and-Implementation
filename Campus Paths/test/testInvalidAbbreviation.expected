Menu:
	r to find a route
	b to see a list of all buildings
	q to quit

Enter an option ('m' to see the menu): # Test the output when invalid abbreviated names are entered
# when getting the shortest path

Abbreviated name of starting building: Abbreviated name of ending building: Unknown building: OOG

Enter an option ('m' to see the menu): Abbreviated name of starting building: Abbreviated name of ending building: Unknown building: WHAT
Unknown building: AH

Enter an option ('m' to see the menu): 